src/ensembl/src/content/app/browser/Browser.test.tsx

```diff
@@ -116,23 +127,28 @@ describe('<Browser />', () => {
         browserQueryParams: { focus: 'foo' }
       };
 
-      it('is rendered when props.browserNavOpened is true', () => {
-        const wrapper = mountBrowserComponent(props);
-        expect(wrapper.find(BrowserNavBar).length).toBe(0);
+      it('is rendered when props.browserNavOpenState is true', () => {
+        let { container } = mountBrowserComponent(props);
+        expect(container.querySelectorAll('.browserNavBar')).toHaveLength(0);
 
-        wrapper.setProps({ browserNavOpened: true });
-        expect(wrapper.find(BrowserNavBar).length).toBe(1);
+        container = mountBrowserComponent({
+          ...props,
+          browserNavOpenState: true
+        }).container;
+        expect(container.querySelectorAll('.browserNavBar')).toHaveLength(1);
       });
 
       it('is not rendered if drawer is opened', () => {
-        const wrapper = mountBrowserComponent({
+        let { container } = mountBrowserComponent({
           ...props,
-          browserNavOpened: true
+          browserNavOpenState: true
         });
-        expect(wrapper.find(BrowserNavBar).length).toBe(1);
 
-        wrapper.setProps({ isDrawerOpened: true });
-        expect(wrapper.find(BrowserNavBar).length).toBe(0);
+        expect(container.querySelectorAll('.browserNavBar')).toHaveLength(1);
+
+        container = mountBrowserComponent({ isDrawerOpened: true }).container;
```

Comment:
Will this test also benefit from rerender?

Reply:
Updated it here and in few other tests above to use `rerender`